tests/README.md

```diff
@@ -1,26 +1,84 @@
 ## Preparing K8S environment for tests
 
+Before preparing K8S environment, make sure you meet the prerequisites 
+defined in [developer guide](https://github.com/openvinotoolkit/operator/blob/main/docs/developer_guide.md). 
+
+1. Install required CLI tools as described in [Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl).
+To install specific version, check current stable version using ```apt policy <package_name>```, e.g. ```apt policy kubectl```.
+
+```
+set -e
+sudo apt-get update
+sudo apt-get install -y apt-transport-https ca-certificates curl gpg
+sudo mkdir -p -m 755 /etc/apt/keyrings
+curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.28/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list
+sudo apt-get update
+sudo apt-get install -y kubelet=1.28.7-1.1 kubeadm=1.28.7-1.1 kubectl=1.28.7-1.1
+sudo apt-mark hold kubelet kubeadm kubectl
+echo "CLI tools installed"
+```
+
+2. Setup containerd
+
+Set up Docker's apt repository as described in [Docker documentation](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository).
+```
+sudo apt-get update
+sudo apt-get install ca-certificates curl
+sudo install -m 0755 -d /etc/apt/keyrings
+sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+sudo chmod a+r /etc/apt/keyrings/docker.asc
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+```
+
+Then, install containerd:
+
+```
+sudo apt remove containerd
+sudo apt update
+sudo apt install containerd.io
+sudo rm /etc/containerd/config.toml
```

Comment:
why is it removed?

Reply:
This is required if any version of containerd was previously installed. I moved it to the cleanup section.